### PR TITLE
other proceeding changes

### PR DESCRIPTION
**Pull request by saravanakumar-palanichamy**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hmcts-kodatala — COMMENTED** [review]
pls check review comments

File: src/main/java/uk/gov/hmcts/reform/bulkscan/constants/BulkScanConstants.java [hunk 1]
```diff
@@ -226,6 +229,31 @@ public final class BulkScanConstants {
...
+    public static final String OTHER_PROCEEDING_TYPE_OF_ORDER_6 =
+            "other_case_proceeding_for_contact_or_resident_order";
+    public static final String OTHER_PROCEEDING_TYPE_OF_ORDER_7 =
+            "other_case_contact_or_residentOrder_withAdoptioNorder";
+    public static final String OTHER_PROCEEDING_TYPE_OF_ORDER_8 =
+            "other_case_childMaintenanceOrder";
+    public static final String OTHER_PROCEEDING_TYPE_OF_ORDER_9 =
+            "other_case_childArrangementOrder";
+    public static final String TYPE_OF_ORDER = "typeOfOrder";
+    public static final String OTHER_PROCEEDINGS_DETAILS_TABLE = "otherProceedingsDetailsTable";
```
[the remark this answers is not in the document]
Comment (saravanakumar-palanichamy) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/constants/BulkScanConstants.java]: done

File: src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100ValidationService.java [hunk 2]
```diff
@@ -262,4 +265,43 @@ public BulkScanValidationResponse validateApplicantAddressFiveYears(
...
+                errors.add(
+                        String.format(
+                                XOR_CONDITIONAL_FIELDS_MESSAGE,
+                                org.apache.commons.lang3.StringUtils.join(
+                                        getTypeOfOrderEnumFields(), ",")));
+            }
+        }
+
+        return errors;
+    }
```
[the remark this answers is not in the document]
Comment (saravanakumar-palanichamy) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100ValidationService.java]: done

**Review by Murshadhaque — APPROVED** [review]

**Review by hmcts-kodatala — APPROVED** [review]
OK

**Pull request merged**